Src/sched.c

```diff
@@ -19,7 +19,7 @@ extern int	lineno, nr_errs, dumptab, xspin, jumpsteps, columns;
 extern int	u_sync, Elcnt, interactive, TstOnly, cutoff;
 extern short	has_enabled, has_priority, has_code, replay;
 extern int	limited_vis, product, nclaims, old_priority_rules;
-extern int	old_scope_rules, scope_seq[128], scope_level, has_stdin;
+extern int	old_scope_rules, scope_seq[256], scope_level, has_stdin;
```

Comment:
If we are changing such magic number anyways, maybe let's name them e.g. via define?

Reply:
Agree - but, it is not mine - this change is merged here from nimble repo before I've started any of my changes - nevertheless, it seems like nimble still work around it, and I think changing that would make further merging of their master harder to handle - anyway - your choice...

Reply:
Understood.